lib/iex/lib/iex/info.ex

```diff
@@ -43,6 +43,18 @@ defimpl IEx.Info, for: Atom do
     [{"Data type", "Atom"}] ++ description ++ specific_info
   end
 
+  defp module?(atom) do
+    case :code.get_object_code(atom) do
+      :error ->
+        Code.ensure_loaded?(atom)
+
+      {^atom, beam, _path} ->
+        info = :beam_lib.info(beam)
+
```

Comment:
```suggestion
```

Reply:
I personally always separate variables and "main body" of the function, but this is up to you.